libtransmission/crypto-utils-polarssl.cc

```diff
@@ -178,6 +180,54 @@ std::optional<tr_sha1_digest_t> tr_sha1_final(tr_sha1_ctx_t raw_handle)
 ****
 ***/
 
+tr_sha256_ctx_t tr_sha256_init(void)
+{
+    api_sha256_context* handle = tr_new0(api_sha256_context, 1);
+
+#if API_VERSION_NUMBER >= 0x01030800
```

Comment:
nit: It's better to leave comment here and there what version this constant maps to.

Reply:
Eh, I copy pasted this from the sha1 stuff. My vote would be to bump the minimum polarssl/mbedtls version and remove the api version check from both places. I bet the constant maps simply to [1.3.8 (changelog)](https://github.com/Mbed-TLS/mbedtls/blob/1dc6848679982dde8b7cd42a547358eb04fed6ca/ChangeLog#L3516) . Version 1.3.8 was released in 2014.

I honestly don't want to do that work.